widgetry/src/widgets/spinner.rs

```diff
@@ -58,38 +77,46 @@ impl Spinner {
             warn!("Spinner current value is out of bounds!");
         }
 
-        let padded_widget = Widget::row(vec![Widget::new(Box::new(Spinner {
+        let mut spinner = Spinner {
             low,
             high,
             current,
 
             up,
             down,
-
+            drawable: Drawable::empty(ctx),
+            outline,
             top_left: ScreenPt::new(0.0, 0.0),
             dims,
-        }))])
-        // pad to consistent height with other input components
-        // Also, something funny going on with the outline here - without this padding
-        // the outline is cropped and looks undesirably thin. I don't understand
-        // why padding would affect the thickness of the outline stroke
-        .padding(EdgeInsets {
-            top: 2.0,
-            left: 1.0,
-            bottom: 2.0,
-            right: 1.0,
-        })
-        .outline(ctx.style().btn_outline.outline);
-
-        // Wrap in another widget so that the caller can apply any additional padding or whatever
-        // they need without blowing away what's necessary for the spinner control to look right.
-        Widget::row(vec![padded_widget])
```

Comment:
The crash was because existing `find` code elsewhere was not expecting the spinner to be double wrapped. I decided to just figure out how to fix the problem without double wrapping.

Reply:
Oh right, the string-based lookup and downcasting is brittle; that downcast won't understand nesting. :)

Aside: This is one of the few places where I'd say borrow checking / lifetimes is causing us to do something pretty gross. It's much simpler if the panel owns all of its child widgets, but then the owner of the panel likely wants to interact with sliders, spinners, etc kind of directly. I briefly experimented with having the app code own sliders and hand borrows to the panel, but it was a mess.